styles/variables.scss

```diff
@@ -3,3 +3,7 @@ $breakpoint: (
         "table": 767px,
         "desktop": 1200px
 );
+
+$basic-red: #C04728;
```

Comment:
Try never use color names in CSS.
Once color scheme may be changed and you will get:

```css
$basic-red: blue;
$basic-white: black;
$basic-yellow: green;
```

How Bootstrap tries to manage it:

<img width="541" alt="Screenshot 2023-09-28 at 09 47 38" src="https://github.com/evgenytryzo/Tyt-naturalnoye-vino-project/assets/496713/f2efdef8-db52-4e66-a451-a4e415c1ed22">

Reply:
Thanks, fixed